### PR TITLE
Remove examples

### PR DESCRIPTION
**Pull request by rfuest**
Thank you for helping out with embedded-graphics-simulator development! Please:

- [ ] Check that you've added passing tests and documentation
- [ ] Add an example where applicable
- [ ] Add a `CHANGELOG.md` entry in the **Unreleased** section under the appropriate heading (**Added**, **Fixed**, etc) if your changes affect the **public API**
- [ ] Run `rustfmt` on the project
- [ ] Run `just build` (Linux/macOS only) and make sure it passes. If you use Windows, check that CI passes once you've opened the PR.

## PR description

This PR removes the examples that were moved to the `examples` repo.

### CODE REVIEW

File: justfile [hunk 1]
```diff
@@ -95,19 +95,3 @@ generate-drawing-examples-montage:
         {{doc_assets_dir}}/display*.png \
         -tile 6x2 -background none -geometry 128x128+4+4 miff:- | \
     convert - -trim {{doc_assets_dir}}/all_drawing_ops.png
-
-# Generates a screenshot of an example
-generate-example-screenshot example:
-    @mkdir -p "{{screenshots_dir}}"
-    # Generating {{example}} screenshot
-    EG_SIMULATOR_DUMP="{{screenshots_dir}}/{{example}}.png" \
```
Comment (jamwaffles) [inline justfile:103]: The `screenshots_dir` declaration at the top could be removed as well.

**Review by jamwaffles — CHANGES_REQUESTED** [review]
Should we add a link to README.md and lib.rs pointing to the new examples repo? Maybe that was planned for another PR after embedded-graphics/examples#3 is merged anyway but worth pointing out either way.

File: justfile (hunk 1 above)
Comment (rfuest) [reply justfile:103]: Good catch. I've also removed the unused `ci_build_image` variable.

**Comment by jamwaffles** [comment]
> Should we add a link to README.md and lib.rs pointing to the new examples repo? Maybe that was planned for another PR after [embedded-graphics/examples#3](https://github.com/embedded-graphics/examples/pull/3) is merged anyway but worth pointing out either way.

Do you have any thoughts on this? Happy to do it in a future PR if that was the plan.

**Comment by rfuest** [comment]
> Do you have any thoughts on this? Happy to do it in a future PR if that was the plan.

Sorry I had somehow missed that comment. That would be a good idea and it would be great if you could create a PR for this.

**Review by jamwaffles — APPROVED** [review]

**Comment by jamwaffles** [comment]
Done in #19

**Pull request merged**